projects/Douglas Mentoring/day1.py

```diff
@@ -0,0 +1,92 @@
+import getpass
+import netmiko
+# json was here bc as I'm using textfsm to pull data from cisco devices
+# it comes in json i needed to show the output before deciding how to tread the date
+import json
+import csv
+
+# Where i'm pulling the devices list / Soon I'll make this automated which will pull direct from NARF
+file = open(r'/home/segattod/segattod-workspace/src/Segattod-package/myproject/Light_level_checks/list.txt',
+            'r')
+file = file.read().splitlines()
+
+username = input('User Name:')
+password = getpass.getpass()
+
+# Variable for "append" the data later
+formatted_data = []
+
+# for now just to name the file / Soon to add for user to add input :)
+whid = 'YEG1'
+
+
+# Function to sorte the data
+def light_level_check(connection):
+    # Command to checks light level per interface
+    transceiver_output = connection.send_command('show interface transceiver', read_timeout=90,
+                                                 use_textfsm='~/ntc-templates/ntc_templates/templates/cisco_ios_show_interface_transceiver.textfsm')
```

Comment:
Python is super flexible on whitespace.  the suggestion below is the same as far as python cares, but way easier for a human to read:

```suggestion
    transceiver_output = connection.send_command(
        'show interface transceiver', 
        read_timeout=90,
        use_textfsm='~/ntc-templates/ntc_templates/templates/cisco_ios_show_interface_transceiver.textfsm'
    )
```